bitrixSM/internal/models/b24/client.go

```diff
@@ -0,0 +1,57 @@
+package b24models
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+type Client struct {
+	WebhookURL string
+	HTTPClient *http.Client
+}
+
+func NewClient(webhookURL string) *Client {
+	return &Client{
+		WebhookURL: webhookURL,
+		HTTPClient: &http.Client{Timeout: 10 * time.Second},
+	}
+}
+
+func (c *Client) CallMethod(method string, params interface{}, result interface{}) error {
```

Comment:
for simple usage it will be ok, but working with interfaces as external interface for your functions is not the best idea usually. It leads to many problems. This is why nobody use vanilla javascript for medium and big production applications!